src/openrct2/interface/widget.c

```diff
@@ -293,12 +293,14 @@ static void widget_tab_draw(rct_drawpixelinfo *dpi, rct_window *w, sint32 widget
 	if (widget->image == -1)
 		return;
 
+	// Draw outline.
+	widget_draw_image(dpi, w, widgetIndex);
```

Comment:
This will break buttons which use a different image for disabled, e.g. the toolbar icons.